sippy-ng/src/App.js

```diff
@@ -436,6 +437,13 @@ export default function App(props) {
                     )}
                   />
 
+                  <Route
+                    path="/prow_job_runs/:jobrunid"
```

Comment:
Was thinking this would be a generic page for details of a job run, perhaps it should be more intervals specific. I can't imagine we'd want anything else on the screen.

Reply:
I could see us having more info about a job run (risk anaylsis maybe?) or eventually even some testgrid like view.

WDYT about `/jobs_runs/:jobrunid/intervals` ?  Just so if we don't break urls later if we need to refactor things. Also do we need prow in the url name?